includes/definitions/marathonups.yaml

```diff
@@ -0,0 +1,9 @@
+os: marathonups
+text: 'MarathonUPS'
+type: power
+icon: marathon
+over:
+    - { graph: device_load, text: Load }
+    - { graph: device_voltage, text: Voltage }
```

Comment:
A lot of (if not all) the sensors you've added can be found if you just add:

`rfc1628_compat: true` to this yaml file. Remove the discovery definition, add that in and try a discovery to see if you get everything.

Reply:
Will do.  Thanks!

Reply:
That worked great.  Thank you!   Nothing like doing it the hard way the first time.  :)